### ISSUE TITLE
BravoTV not working

### ISSUE THREAD

**Issue by robojerk** [yt-dlp/yt-dlp#6997]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that a **supported** site is broken
- [X] I've verified that I'm running yt-dlp version **2023.03.04** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

USA

### Provide a description that is worded well enough to be understood

I get an error when attempting to download from BravoTV

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', 'https://www.bravotv.com/last-chance-kitchen/season-20/videos/who-ya-rooting-for']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.03.04 [392389b7d]
[debug] Lazy loading extractors is disabled
[debug] Python 3.11.3 (CPython x86_64 64bit) - Linux-6.2.12-200.fsync.fc37.x86_64-x86_64-with-glibc2.36 (OpenSSL 3.0.8 7 Feb 2023, glibc 2.36)
[debug] exe versions: ffmpeg 5.1.3 (setts), ffprobe 5.1.3
[debug] Optional libraries: Cryptodome-3.17.0, brotli-1.0.9, certifi-2021.10.08, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] Loaded 1786 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.03.04, Current version: stable@2023.03.04
yt-dlp is up to date (stable@2023.03.04)
[BravoTV] Extracting URL: https://www.bravotv.com/last-chance-kitchen/season-20/videos/who-ya-rooting-for
[BravoTV] who-ya-rooting-for: Downloading webpage
ERROR: who-ya-rooting-for: An extractor error has occurred. (caused by KeyError('account_pid')); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "/usr/lib/python3.11/site-packages/yt_dlp/extractor/common.py", line 694, in extract
    ie_result = self._real_extract(url)
                ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/yt_dlp/extractor/bravotv.py", line 74, in _real_extract
    account_pid = shared_playlist['account_pid']
                  ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
KeyError: 'account_pid'
```

**Comment by bashonly** [comment]
This has been [fixed in master](https://github.com/yt-dlp/yt-dlp/commit/06966cb8966b9aa4f60ab9c44c182a057d4ca3a3)

[Get the nightly build](https://github.com/yt-dlp/yt-dlp-nightly-builds/releases/latest) or [install from master branch](https://github.com/yt-dlp/yt-dlp/wiki/Installation#with-pip)

Closing as duplicate of #6562

**Comment by bossanova808** [comment]
I'm still seeing this with the latest nightly:
```
yt-dlp --version
2023.06.17.224306
...
[BravoTV] Extracting URL: https://www.bravotv.com/last-chance-kitchen/season-20/videos/what-a-grind
[BravoTV] what-a-grind: Downloading webpage
ERROR: what-a-grind: An extractor error has occurred. (caused by KeyError('account_pid')); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "/home/xx/.local/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 694, in extract
    ie_result = self._real_extract(url)
  File "/home/xx/.local/lib/python3.10/site-packages/yt_dlp/extractor/bravotv.py", line 74, in _real_extract
    account_pid = shared_playlist['account_pid']
KeyError: 'account_pid'
```

**Comment by bashonly** [comment]
```
  File "/home/xx/.local/lib/python3.10/site-packages/yt_dlp/extractor/bravotv.py", line 74, in _real_extract
    account_pid = shared_playlist['account_pid']
```
that's not nightly. that's a pip installation, or some other package manager installation

`account_pid` was made non-fatal in the patch. not possible that you are actually running nightly and getting that error

**Comment by bossanova808** [comment]
I might be wrong, but looks like nightly to me?

```
sudo yt-dlp --update-to nightly
Available version: nightly@2023.06.17.224306, Current version: nightly@2023.06.17.224306
Current Build Hash: 6774b889af8aec948511256bde94834063bb4729ea1e0a89a5b79e38a71e3593
yt-dlp is up to date (nightly@2023.06.17.224306)
```
hmmm

No you're right.  Looks like I have an older version too.  Apologies for the noise.

```
pip uninstall yt-dlp
Found existing installation: yt-dlp 2023.3.4
Uninstalling yt-dlp-2023.3.4:
  Would remove:
    /home/jeremy/.local/lib/python3.10/site-packages/yt_dlp-2023.3.4.dist-info/*
    /home/jeremy/.local/lib/python3.10/site-packages/yt_dlp/*
    /home/jeremy/.local/share/bash-completion/completions/yt-dlp
    /home/jeremy/.local/share/doc/yt_dlp/README.txt
    /home/jeremy/.local/share/fish/vendor_completions.d/yt-dlp.fish
    /home/jeremy/.local/share/man/man1/yt-dlp.1
    /home/jeremy/.local/share/zsh/site-functions/_yt-dlp
Proceed (Y/n)? Y
  Successfully uninstalled yt-dlp-2023.3.4
```

Now I just have to work out how I can have the nightly _and_ call it (embedded) from python...is that possible?

**Comment by gamer191** [comment]
@bossanova808 see point 3 https://github.com/yt-dlp/yt-dlp/wiki/Installation#with-pip
